ogc-example/src/main/scala/geotrellis/server/ogc/Main.scala

```diff
@@ -106,20 +106,44 @@ object Main extends CommandApp(
               .values
               .collect { case ssc@SimpleSourceConf(_, _, _, _) => ssc.model }
               .toList
-            wmsModel = WmsModel(
-              conf.wms.serviceMetadata,
-              conf.wms.parentLayerMeta,
-              conf.wms.layerSources(simpleSources)
-            )
-            wmtsModel = WmtsModel(
-              conf.wmts.serviceMetadata,
-              conf.wmts.tileMatrixSets,
-              conf.wmts.layerSources(simpleSources)
-            )
-            wcsModel = WcsModel(
-              conf.wcs.serviceMetadata,
-              conf.wcs.layerSources(simpleSources)
-            )
+            _ <- Stream.eval(IO {
```

Comment:
I was confused for a second as to why this block was here, but its just so that we get logger output on both cases. 

Since each of these blocks are coupled to the ones immediately following, would a `conf.wms.fold()()` work for each of these? In the default case you can log and pass through None, in the valid case you could add the logger statement to the existing map block.

I don't feel strongly about it, just thinking of ways to try to merge these two blocks in a sensible way.

Reply:
:+1: to using `conf.wms.fold` instead of `conf.wms.isDefined`

Reply:
I believe this change makes things less readable 🤷‍♂

Reply:
Bah. Yeah, I agree with you. Sorry about the poor suggestion -- it was more of musing level of comment.